pkg/controller/direct/logging/utils.go

```diff
@@ -130,7 +192,16 @@ func convertAPItoKRM_LogMetricLabels(apiLabels []*api.LabelDescriptor) []krm.Log
 			Key:         &apiLabel.Key,
 			ValueType:   &apiLabel.ValueType,
 		}
+		// TODO(acpana): this looks like a bug on the api, as the value should be defaulted already.
```

Comment:
I think I've seen this from the API, where `STRING` gets returned from the API as `""`.  I would call it a "quirk" of the API :-)  I bet if you look at the proto, it's an enum, and STRING == 0.